docs/en/deployment/production_deployment_recommendations.md

```diff
@@ -5,4 +5,86 @@ slug: /production_deployment_recommendations
 
 # Production Deployment Recommendations
 
-TBD
+This article aims to give some recommendations when deploying JuiceFS to a production environment, so please read the following in advance and carefully.
+
+## Metrics Collection and Visualization
+
+Be sure to collect and visualize the monitoring metrics for your JuiceFS client through Grafana, as described in [documentation](... /administration/monitoring.md).
+
+## Automatic Metadata Backup
+
+:::tip
+Automatic metadata backup is a feature that has been added since JuiceFS v1.0.0
+:::
+
+Metadata is critical to JuiceFS file system, any loss or corruption may affect a large number of files or even the entire file system. Therefore, metadata must be backed up regularly.
+
+This feature is enabled by default and the backup interval is 1 hour. The backup metadata will be compressed and stored in the corresponding object storage (isolated from the file system data). Backups are performed by JuiceFS clients, which may cause an increase in CPU and memory usage during backups, and by default it is assumed that one client will be randomly selected to perform the operation.
+
+Note in particular that this feature will be turned off when the number of files reaches **one million**. To turn it on again, you have to set a larger backup interval (the `--backup-meta` option) . The interval is configured independently for each client, and setting `--backup-meta 0` disables the automatic backup.
+
+:::note
+The time required to back up metadata depends on the specific metadata engine, and different metadata engines will have different performance.
+:::
+
+For a detailed description of automatic metadata backups, please refer to [documentation](. /administration/metadata_dump_load.md#Automatic Backup), or you can back up metadata manually. In addition, please also follow the O&M recommendations of the metadata engine you are using to perform regular backups of your data.
+
+## Trash
+
+:::tip
+The Trash is a feature added since JuiceFS v1.0.0
+:::
+
+Trash is enabled by default, and the retention time for deleted files is 1 day by default, which can effectively prevent the risk of loss when data is deleted by mistake.
+
+However, when the trash is enabled, it may also bring some side effects. If the application needs to frequently delete files or overwrite them, it will cause the object storage usage to be much larger than that of file system. This is because the JuiceFS client will keep deleted files and overwritten blocks on the object storage for a period of time. Therefore, when deploying JuiceFS to a production environment you'd better evaluate the workload to get a proper trash configuration. The retention time can be configured in the following way (`--trash-days 0` means disable trash)
```

Comment:
```suggestion
However, when the trash is enabled, it may also bring some side effects. If the application needs to frequently delete files or overwrite them, it will cause the object storage usage to be much larger than the file system. This is because the JuiceFS client will keep deleted files and overwritten blocks on the object storage for a period of time. Therefore, when deploying JuiceFS to a production environment, it is highly recommended to evaluate the workload ahead to get a proper trash configuration. The retention time can be configured in the following way (`--trash-days 0` means to disable trash)
```